Zune.Net.Shared/Helpers/Deezer.Album.cs

```diff
@@ -1,16 +1,17 @@
-﻿using MetaBrainz.MusicBrainz.Interfaces.Entities;
+﻿using MetaBrainz.MusicBrainz.Interfaces;
+using MetaBrainz.MusicBrainz.Interfaces.Entities;
 using Newtonsoft.Json.Linq;
 using System.Linq;
 
 namespace Zune.Net.Helpers
 {
     public partial class Deezer
     {
-        public static IRelease GetMBReleaseByDZAlbum(JToken dz_album)
+        public static IReleaseGroup GetMBReleaseGroupByDZAlbum(JToken dz_album)
         {
             JToken dz_artist = dz_album["artist"];
-            var results = MusicBrainz._query.FindAllReleases(
-                $"artistname:{dz_artist.Value<string>("name")} AND release:{dz_album.Value<string>("title")}", simple: false);
+            IStreamingQueryResults<MetaBrainz.MusicBrainz.Interfaces.Searches.ISearchResult<IReleaseGroup>>results = MusicBrainz._query.FindAllReleaseGroups(
```

Comment:
Is there a reason to write the variable type here, rather than leaving it as `var`?

Reply:
I was always told it's better to declare variables by type to ensure that if something is returned as an incorrect object type, that the debugger errors out and is found.